src/main/kotlin/prison/workspace.kt

```diff
@@ -12,10 +14,41 @@ fun prisonWorkspace(): Workspace {
     setId(55246)
   }
 
-  cloudPlatform(workspace.model)
+  // The systems we wish to include in this workspace
+  val systems = listOf<HMPPSSoftwareSystem>(
+    NOMIS,
+    PrisonerContentHub
+  )
+
+  // We start by defining the deployment nodes. Containers are often associated with deployment nodes
+  // during model definition, so we front-load this task to make everything available for that step.
+  CloudPlatform.defineDeploymentNodes(workspace.model)
```

Comment:
What do you think about removing the comments? They have a nasty habit of getting out of date quickly.

Reply:
Comments feel useful for explaining the intent of the code, and here it feels like they're useful to convey the importance of ordering. 

I would _hope_ that if someone changes the order or introduces other dependencies, either they'll update these comments or we'd catch it in a pull request, but I'm not sure I agree that it's better _not_ to have them in case they go out of date 🤷‍♂️.

Reply:
Maybe it's possible to make that implicit ordering clearer with the names 🤔